src/shitalSDec19/assignment3/Solution6.java

```diff
@@ -0,0 +1,39 @@
+/* WAP to find reverse max length string 
+a. Input array:- {�all the best�, �technocredits�, ��, �maharashtra�} 
+b. Output Max length string :- Technocredits 
+c. Max length string reverse value :- stiderconhcet  */
+package shitalSDec19.assignment3;
+
+public class Solution6 {
+	
+	void findMaxLengthString(String [] arr)
+	{ 
+		String b="";
+		String reverse = "";
+		int maxLength=0;
+		for (int i=0; i<arr.length; i++)
+		{
+			String s = arr[i];
+			String [] a = s.split(",");
```

Comment:
what is use of split over here?

Reply:
I was trying to make use of split but it was of no use. I have updated my program but still i am unable to print expected reverse string. Please review.